HMTL documentation for ConErrSkip option.

The patch was contributed to upstream and currently it is accepted:
https://github.com/rsyslog/rsyslog-doc/pull/916

--- rsyslog-doc-8.2002.0/build/configuration/modules/omfwd.html 14:28:28.727329028 +0100
+++ rsyslog-doc-8.2002.0/build/configuration/modules/omfwd.html 15:16:57.889080346 +0100
@@ -569,6 +569,30 @@
 This has only effect if keep-alive is enabled. The functionality may
 not be available on all platforms.</p>
 </div>
+<div class="section" id="conerrskip">
+<h4>ConErrSkip<a class="headerlink" href="#conerrskip" title="Permalink to this headline">¶</a></h4>
+<table border="1" class="colwidths-auto parameter-table docutils">
+<thead valign="bottom">
+<tr class="row-odd"><th class="head">type</th>
+<th class="head">default</th>
+<th class="head">mandatory</th>
+<th class="head"><code class="docutils literal"><span class="pre">obsolete</span> <span class="pre">legacy</span></code> directive</th>
+</tr>
+</thead>
+<tbody valign="top">
+<tr class="row-even"><td>integer</td>
+<td>0</td>
+<td>no</td>
+<td>none</td>
+</tr>
+</tbody>
+</table>
+<p>The conerrskip can be used to limit the number of network errors
+recorded in logs. For example, value 10 means that each 10th error
+message is logged. Note that this options should be used as the last
+resort since the necessity of its use indicates network issues.
+The default behavior is that all network errors are logged.</b>
+</div>
 <div class="section" id="streamdriver">
 <h4>StreamDriver<a class="headerlink" href="#streamdriver" title="Permalink to this headline">¶</a></h4>
 <table border="1" class="colwidths-auto parameter-table docutils">
